host/crypto/bcrypt/rsa.c

```diff
@@ -321,7 +321,7 @@ oe_result_t oe_rsa_public_key_get_modulus(
     oe_result_t result = OE_UNEXPECTED;
     const oe_public_key_t* impl = (const oe_public_key_t*)public_key;
     BYTE* key_blob = NULL;
-    ULONG key_blob_size;
+    ULONG key_blob_size = 0;
     BCRYPT_RSAKEY_BLOB* keyblob;
```

Comment:
Should this (and other `BCRYPT_RSAKEY_BLOB*`s in this file) be initialized to `NULL`?